solutions/009_Find_the_Runner_Up_Score.md

```diff
@@ -106,3 +106,19 @@ arr = list(arr)
 new_arr = remove_n(arr, max(arr))
 print(max(new_arr))
 ```
+
+##Solution 5
+...
+if __name__ == '__main__':
+    n = int(input())
+    arr = map(int, input().split())
+    l=[]
+    l=list(arr)
```

Comment:
Thank you for your contribution! Your solution is correct, but we can make a few improvements. You define `l` twice as `list` you don't need that. In fact, doing as in the example below where you define `arr` can further simplify your code.

`arr = list(map(int, input().split()))`